[zh-cn] update django local library tutorial part 1

jasonren0403 opened the pull request:
<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

<!-- ✍️ Summarize your changes in one or two sentences -->

### Motivation

<!-- ❓ Why are you making these changes and how do they help readers? -->

### Additional details

<!-- 🔗 Link to release notes, vendor docs, bug trackers, source control, or other places providing more context -->

### Related issues and pull requests

<!-- 🔨 If this fully resolves a GitHub issue, use "Fixes #123" -->
<!-- 👉 Highlight related pull requests using "Relates to #123" -->
<!-- ❗ If another pull request should be merged first, use "**Depends on:** #123" -->

<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

jasonren0403 committed:
[zh-cn] update django local library tutorial part 1

jasonren0403 commented on files/zh-cn/learn_web_development/extensions/server-side/django/tutorial_local_library_website/index.md:
```diff
@@ -30,43 +33,43 @@ slug: Learn_web_development/Extensions/Server-side/Django/Tutorial_local_library
 - 接下来，图书馆会慢慢扩展来展示更高级的 Django 特性。例如，我们会扩展功能，让会员能够保留图书。这个特性会展示如何使用表单，并支持用户认证。
 
-尽管这是一个非常容易扩展的例子，它被称为本地图书馆是有原因的——我们希望用最少的信息帮助你快速创建和运用 Django。最后，我们会存储图书信息，图书数量，作者和其他重要信息。我们不会存储图书馆可能会存储的其他信息，或是提供一个支持多个图书馆或是”大型图书馆“功能的构建。
+尽管这是一个非常容易扩展的例子，它被称为***本地**图书馆*是有原因的——我们希望用最少的信息帮助你快速创建和运用 Django。最后，我们会存储图书信息，图书数量，作者和其他重要信息。我们不会存储图书馆可能会存储的其他信息，或是提供一个支持多个图书馆或是“大型图书馆”功能的构建。
```
又是一个斜体套黑体的例子 🤔 
![image](https://github.com/user-attachments/assets/196f1c9a-c6c8-4f68-aa0f-b1f746e03c39)

yin1999 replied:
昨晚我试了一下[这里](https://github.com/mdn/translated-content/pull/27050#discussion_r2078726683)提到的修复后的 prettier 版本，好像还是用不了。不行的话还得去 prettier 提新的修复。我不确定是不是我用的方法不对：

```bash
yarn add https://pkg.pr.new/prettier@17143

yarn prettier -w /path/to/file.md
```

jasonren0403 committed:
Merge branch 'main' into zhcn-django-locallibrary-1

yin1999 committed:
Merge branch 'main' into zhcn-django-locallibrary-1

yin1999 committed:
Merge branch 'main' into zhcn-django-locallibrary-1

yin1999 committed:
sync content

Updated the source commit reference and revised the introduction to the tutorial.

yin1999 committed:
Update index.md

yin1999 committed:
fix format

yin1999 committed:
minor

yin1999 reviewed (approved):
做了一下兼容，这里的格式和其他加粗加斜体的格式又不太一样，prettier里面可能没那么好改了，先做个保底更新上去吧
